api/util/util.py

```diff
@@ -11,6 +11,7 @@ def decorator(*args, **kwargs):
         t = Thread(target=function, args=args, kwargs=kwargs)
         t.daemon = True
         t.start()
+        return t
```

Comment:
Maybe I missed it, but I'm not sure where/if this is used? If it is used, we should add it to the typing for the function.

Reply:
Not used; removed this.